src/neo/SmartContract/Native/Tokens/GasToken.cs

```diff
@@ -44,30 +37,7 @@ protected override bool OnPersist(ApplicationEngine engine)
             ECPoint[] validators = NEO.GetNextBlockValidators(engine.Snapshot);
             UInt160 primary = Contract.CreateSignatureRedeemScript(validators[engine.Snapshot.PersistingBlock.ConsensusData.PrimaryIndex]).ToScriptHash();
             Mint(engine, primary, engine.Snapshot.PersistingBlock.Transactions.Sum(p => p.NetworkFee));
-            BigInteger sys_fee = GetSysFeeAmount(engine.Snapshot, engine.Snapshot.PersistingBlock.Index - 1) + engine.Snapshot.PersistingBlock.Transactions.Sum(p => p.SystemFee);
-            StorageKey key = CreateStorageKey(Prefix_SystemFeeAmount, BitConverter.GetBytes(engine.Snapshot.PersistingBlock.Index));
-            engine.Snapshot.Storages.Add(key, new StorageItem
-            {
-                Value = sys_fee.ToByteArrayStandard(),
-                IsConstant = true
-            });
             return true;
         }
-
-        [ContractMethod(0_01000000, ContractParameterType.Integer, ParameterTypes = new[] { ContractParameterType.Integer }, ParameterNames = new[] { "index" }, SafeMethod = true)]
-        private StackItem GetSysFeeAmount(ApplicationEngine engine, VMArray args)
-        {
-            uint index = (uint)args[0].GetBigInteger();
-            return GetSysFeeAmount(engine.Snapshot, index);
-        }
-
-        public BigInteger GetSysFeeAmount(StoreView snapshot, uint index)
-        {
-            if (index == 0) return Blockchain.GenesisBlock.Transactions.Sum(p => p.SystemFee);
```

Comment:
Should we keep the `SysFeeAmount` ? If we want to increase the circulation of gas, will it depend on the `SysFeeAmount`?